setup.py

```diff
@@ -18,8 +18,8 @@
     url='https://github.com/takeflight/wagtailvideos',
 
     install_requires=[
-        'wagtail>=1.7',
-        'Django>=1.9',
+        'wagtail>=2.0',
+        'Django>=2.0',
```

Comment:
Wagtail 2.0 is still compatible with Django 1.11, it would be nice to keep this available if possible.